store/postgres/src/relational.rs

```diff
@@ -1436,6 +1439,11 @@ pub struct Table {
     /// aggregations, this is the object type for a specific interval, like
     /// `Stats_hour`, not the overall aggregation type `Stats`.
     pub object: EntityType,
+
+    /// The name of the database table for this type ('thing'), snakecased
+    /// version of `object`
+    pub namespace: SqlName,
```

Comment:
I don't think this is needed; the one place where this is read is in `Table::as_ddl` where you have a `Catalog` and can get the namespace from its `site`. If it truly is needed, the comment needs to be adjusted.

Reply:
Yes, it's not needed. The value in catalog is the same. Removed it.